### PR TITLE
Simple test to verify empty entries in white-/black-lists are handled correctly

### PR DESCRIPTION
**Pull request by chavdar**
* Refactor EtlInputFormat.getKafkaWhitelistTopic()/getKafkaBlacklistTopic() to simplify testing
* Add a simple test to verify empty entries in whitelists/blacklists are handled correctly. This is important to be able to handle cases like:

kafka.blacklist.topics=${blacklist.topics.A},${blacklist.topics.B}
blacklist.topics.A=
blacklist.topics.B=TopicB1,TopicB2

### CODE REVIEW

**Comment by zliu41** [comment]
LGTM

File: camus-etl-kafka/src/test/java/com/linkedin/camus/etl/kafka/mapred/EtlInputFormatTest.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.linkedin.camus.etl.kafka.mapred;
+
+import java.util.Arrays;
+
+import org.apache.hadoop.conf.Configuration;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class EtlInputFormatTest {
```
Comment (HenryCaiHaiying) [inline camus-etl-kafka/src/test/java/com/linkedin/camus/etl/kafka/mapred/EtlInputFormatTest.java]: We actually just created this class 'EtlInputFormatTest', you can either combine your code with that class or rename your class.

**Comment by HenryCaiHaiying** [comment]
For the use case mentioned below, is this the existing case or future one?

kafka.blacklist.topics=${blacklist.topics.A},${blacklist.topics.B}
blacklist.topics.A=
blacklist.topics.B=TopicB1,TopicB2

**Comment by chavdar** [comment]
@HenryCaiHaiying This is a future use-case.

I've merged the upstream changes. All unit tests pass:

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] Camus Parent ....................................... SUCCESS [  0.418 s]
[INFO] Camus API. ......................................... SUCCESS [  1.262 s]
[INFO] Camus schema registry. ............................. SUCCESS [  0.993 s]
[INFO] Camus Kafka producer ............................... SUCCESS [  0.849 s]
[INFO] Camus ETL to move data from Kafka to Hadoop. ....... SUCCESS [ 49.239 s]
[INFO] Camus schema registry. ............................. SUCCESS [  2.933 s]
[INFO] Camus Examples ..................................... SUCCESS [  0.977 s]
[INFO] Camus compaction code small files produced Camus ... SUCCESS [  0.116 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
